pypit/tests/files/LRISb_600_WaveCalib_01.json

```diff
@@ -0,0 +1,2139 @@
+{
```

Comment:
For a unit test, it might be more desirable to have a more compact (smaller) file, especially once we start adding many more unit tests.